### PR TITLE
Create odd_or_even.py

### PR DESCRIPTION
**Pull request by vicky1999**
Created odd_or_even.py to find whether a number is odd or even using bit manipulation.

For any odd number n: (n&1)=1
For any Even number n: (n&1)=0

### CODE REVIEW

**Comment by vicky1999** [comment]
===================short test summary info==========================
FAILED bit_manipulation/odd_or_even.py::odd_or_even.odd_or_even
============ 1 failed, 594 passed, 776 warnings in 75.70s (0:01:15) ============
The command "pytest --doctest-modules --cov-report=term-missing:skip-covered --cov=. ." exited with 1.

File: bit_manipulation/odd_or_even.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+def odd_or_even(n: int) -> str:
+    """
+    whether a given number is odd or even using bitwise AND
+    >>> odd_or_even(1)
+    Odd
+    >>> sum_of_digits(14)
```
Comment (poyea) [inline bit_manipulation/odd_or_even.py:6]: What's `sum_of_digits`?

Comment (cclauss) [reply bit_manipulation/odd_or_even.py:6]: Copy/paste error from https://github.com/TheAlgorithms/Python/blob/master/maths/sum_of_digits.py

File: bit_manipulation/odd_or_even.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    Even
+    """
+    if(n & 1 == 0):
+        return "Odd"
+    else:
+        return "Even"
```
Comment (cclauss) [inline bit_manipulation/odd_or_even.py:12]:
```suggestion
    return "Odd" if n % 1 else "Even"
```

**Comment by cclauss** [comment]
https://travis-ci.com/github/TheAlgorithms/Python/builds/169456024#L748

File: bit_manipulation/odd_or_even.py (hunk 1 above)
Comment (vicky1999) [reply bit_manipulation/odd_or_even.py]: Now I have changed the code and documentations

File: bit_manipulation/odd_or_even.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    >>> odd_or_even(1)
+    Odd
+    >>> odd_or_even(14)
+    Even
+    """
+    res = n & 1
+    if(res == 0):
+        return "Odd"
+    else:
+        return "Even"
```
Comment (TheSuperNoob) [inline bit_manipulation/odd_or_even.py:13]: This can be simplified to:

`return "Odd" if n & 1 else "Even" `

**Pull request closed without merging**